omod/src/main/webapp/fragments/infoAndErrorMessages.gsp

```diff
@@ -1,5 +1,7 @@
 <% if (errorMessage) { %>
-    <div id="error-message">${ errorMessage }</div>
+    <div class="alert alert-danger">
```

Comment:
What happens when you remove the div on line 2?

Reply:
@dkayiwa dropping div 2 drops the  red back- grounded section in which we have the error message,should we go by that?,thanks

Reply:
@HerbertYiga So what if we had this instead:

```suggestion
    <div id="error-message" class="alert alert-danger" role="alert"><i class="icon-exclamantion-sign"></i>${ errorMessage }</div>
```

Reply:
thanks @ibacher  let me try that and drop the screen shot on the ticket Id

Reply:
@dkayiwa  @ibacher  made changes on this as suggested by @ibacher ,added a screen shot on the ticket Id named (latest screen shot) and this appears as the initial ones.thanks

Reply:
@ibacher  hi ian,does this look good for merging?